caffe2/CMakeLists.txt

```diff
@@ -1044,6 +1044,9 @@ if(USE_XPU)
   add_library(torch_xpu ${Caffe2_XPU_SRCS})
   torch_compile_options(torch_xpu)  # see cmake/public/utils.cmake
   target_compile_definitions(torch_xpu PRIVATE USE_XPU)
+  if(WIN32)
+    target_compile_options(torch_xpu PRIVATE /permissive-)
```

Comment:
We added the option `/permissive-` to ask the compiler to disable permissive behaviors and execute strict conformance to avoid ambiguous symbol error on Windows. Please refer to [`/permissive-`](https://learn.microsoft.com/en-us/cpp/build/reference/permissive-standards-conformance?view=msvc-170) for more details.